### PR TITLE
Fix capitalisation in tutorial

### PR DESCRIPTION
**Pull request by regular-pencil**
## Description
This is a test PR from codespaces to see if I can raise a PR

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/docs/pages/tutorials/first-dashboard.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 There is no setup needed for your first dashboard, thanks to the amazing [Py.Cafe](https://py.cafe/).
 
-Click on the **Run and edit this code in Py.Cafe** link below to live-edit the dashboard.
+Click on the **Run and edit this code in py.cafe** link below to live-edit the dashboard.
```
Comment (regular-pencil) [inline vizro-core/docs/pages/tutorials/first-dashboard.md:5]: This is such an amazing change. I must be a genius

Comment (stichbury) [reply vizro-core/docs/pages/tutorials/first-dashboard.md:5]: Do you have insight onto the style guide for use of "py.cafe" vs "Py.Cafe" that I don't? I have noticed they're calling it [`Py.cafe` here](https://py.cafe/terms) but [`py.cafe` in the ToS](https://app.termly.io/policy-viewer/policy.html?policyUUID=6a4b19ee-881e-4940-867c-3bc978e5ec84) and [`PyCafe` on the About page](https://py.cafe/about). We've consistently used `Py.Cafe` throughout and I think that is definitely wrong.

Maybe a GHC ticket to correct it to "PyCafe" where we are talking about the organisation and "py.cafe" when talking specifically about "navigating to py.cafe" that is, the website.

Comment (regular-pencil) [reply vizro-core/docs/pages/tutorials/first-dashboard.md]: thanks, as revealed on Slack, this PR was from me (Max) :P

Comment (maartenbreddels) [reply vizro-core/docs/pages/tutorials/first-dashboard.md]: Good question, we prefer to use PyCafe, or the link https://py.cafe

Comment (stichbury) [reply vizro-core/docs/pages/tutorials/first-dashboard.md]: Thanks, i'll note that and create an issue to fix anything else.

**Pull request closed without merging**